cmd/truss/main.go

```diff
@@ -363,11 +363,18 @@ func readPreviousGeneration(serviceDir string) (map[string]io.Reader, error) {
 
 	addFileToFiles := func(path string, info os.FileInfo, err error) error {
 		if info.IsDir() {
-			return nil
+			switch info.Name() {
+			// Only files within the handlers dir are used to
+			// support regeneration.
+			// See `gengokit/generator/gen.go:generateResponseFile`
+			case filepath.Base(serviceDir), "handlers":
```

Comment:
Consider extracting `"handlers"` as a constant at the top of this function.

I wish things were organized in such a way that this directory name was exposed, though such is life.

Reply:
Extracted into a const